### PR TITLE
[TSOPS-1406] Add TSOPS-1406

### PR DESCRIPTION
**Pull request by patrickhargett**
Add 'kubectl cluster-info dump' to the support bundle.

### CODE REVIEW

**Review by dthomson-sysdig — APPROVED** [review]

File: support_bundle/get_support_bundle.sh [hunk 1]
```diff
@@ -195,6 +195,9 @@ main() {
     echo "Using namespace ${NAMESPACE}"
     echo "Using context ${CONTEXT}"
 
+    # Collect kubectl cluster dump
+    kubectl ${KUBE_OPTS} cluster-info dump >> ${LOG_DIR}/kubectl-cluster-dump.json
```
Comment (gmarraff) [inline support_bundle/get_support_bundle.sh:199]: I gave a try to this command (I didn't know it, thanks for making me learn :grin: ) and is not only printing json structures but also al the logs of the containers in `kube-system` and `default` namespaces.
If you specify the `--output-directory` params it also create a nice hierarchy in the specified directory.

So I would suggest to do something like:
```suggestion
    CLUSTER_DUMP_DIR="${LOG_DIR}/kubectl-cluster-dump"
    mkdir ${CLUSTER_DUMP_DIR}
    kubectl ${KUBE_OPTS} cluster-info dump --output-directory=${CLUSTER_DUMP_DIR}
```

But please, feel free to ignore this comment is unnecessary!

Comment (dthomson-sysdig) [reply support_bundle/get_support_bundle.sh:199]: @patrickhargett I like this idea as well from @gmarraff

**Review by gmarraff — APPROVED** [review]

**Review by dthomson-sysdig — DISMISSED** [review]

File: support_bundle/get_support_bundle.sh [hunk 2]
```diff
@@ -195,6 +195,11 @@ main() {
     echo "Using namespace ${NAMESPACE}"
     echo "Using context ${CONTEXT}"
 
+    # Collect kubectl cluster dump
+    CLUSTER_DUMP_DIR="${LOG_DIR}/kubectl-cluster-dump"
+    mkdir ${CLUSTER_DUMP_DIR}
```
Comment (mbreitung) [inline support_bundle/get_support_bundle.sh:200]: please add `-p`

**Review by mbreitung — CHANGES_REQUESTED** [review]

File: support_bundle/get_support_bundle.sh (hunk 2 above)
Comment (patrickhargett) [reply support_bundle/get_support_bundle.sh]: Added!

**Review by dthomson-sysdig — APPROVED** [review]

**Review by gmarraff — APPROVED** [review]

**Review by mbreitung — APPROVED** [review]

**Pull request merged**